python/default_azure_mgmt_credential.py

```diff
@@ -0,0 +1,25 @@
+from azure.identity import DefaultAzureCredential
+from azure.common.credentials import ServicePrincipalCredentials
+import os
+import time
+
+
+class DefaultAzureMgmtCredential(ServicePrincipalCredentials):
+    scopes = 'https://management.azure.com/.default'
+    accessTokenCache = {}
+
+    def __init__(self, client_id=None, secret=None, credentail=None, **kwargs):
```

Comment:
- You shouldn't expose client_id/secret, it brings no value
- credentail is a typo
- scopes should be available as kwargs to override